ruby/overrides.js

```diff
@@ -1,54 +1,29 @@
 module.exports = {
-    // selenium3 pass / 4 fail
-    // CheckRegionBySelectorInFrameFullyOnFirefoxLegacy
-    'check region by selector in frame fully on firefox legacy': {skip:true},
-    // CheckWindowAfterManualScrollOnSafari11
-    'check window after manual scroll on safari 11': {skip:true},
-    // CheckWindowAfterManualScrollOnSafari12
-    // selenium-4 caps: {browserName: "safari", browserVersion: '12', platformName: 'macOS 10.13', :"sauce:username" => "...", :"sauce:accessKey" => "..." },
-    // 'check window after manual scroll on safari 12': {skip:true},
+    // fails in selenium4 only due to legacy driver being used
+    'check window after manual scroll on safari 11': {skip: true},
 
-    // Shadow dom JS errors
-    // CheckRegionByElementWithinShadowDomWithVg
+    // fails on chrome>=96
     'check region by element within shadow dom with vg': {skip: true},
-    // CheckRegionBySelectorWithinShadowDomWithVg
-    // 'check region by selector within shadow dom with vg': {skip: true},
 
-    // Applitools::DiffsFoundError
-    // CheckWindowFullyOnAndroidChromeEmulatorOnDesktopPage - to skip
-    // 'check window fully on android chrome emulator on desktop page': {skip: true},
-    // CheckRegionBySelectorFullyOnPageWithStickyHeaderWithScrollStitching - baseline need update
-    // 'check region by selector fully on page with sticky header with scroll stitching': {skip: true},
-    // CheckWindowFullyOnPageWithStickyHeaderWithScrollStitching - baseline need update
-    // 'check window fully on page with sticky header with scroll stitching': {skip: true},
-    // CheckRegionInFrameHiddenUnderTopBarFullyWithScrollStitching - generation need { type: :css, selector: 'body' }
-    // 'check region in frame hidden under top bar fully with scroll stitching': {skip: true},
-
-    // ShouldHandleCheckOfStaleElementIfSelectorIsPreserved
+    // Stale element are not handled by python binding
     'should handle check of stale element if selector is preserved': {skip: true},
-    // ShouldHandleCheckOfStaleElementInFrameIfSelectorIsPreserved
     'should handle check of stale element in frame if selector is preserved': {skip: true},
+    // OCR is broken again
+    "should extract text regions from image": {skip: true},
+
+    // Different ruby getAllTestResults returned value
+    'should send agentRunId': {skip: true},
+    'should abort after close': {skip: true},
+	'should abort unclosed tests': {skip: true},
+    'should abort unclosed tests with vg': {skip: true},
+    'should return aborted tests in getAllTestResults': {skip: true},
+    'should return aborted tests in getAllTestResults with vg': {skip: true},
+    'should return browserInfo in getAllTestResults': {skip: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9334725)
```suggestion
    "should return browserInfo in getAllTestResults": {skip: true},
```